src/bgem/gmsh/gmsh.py

```diff
@@ -590,8 +590,8 @@ def _assign_physical_groups(self, obj):
 
         # set physical groups
         for reg, tags in reg_to_tags.values():
-            reg._gmsh_id = gmsh.model.addPhysicalGroup(reg.dim, tags, tag=-1)
-            gmsh.model.setPhysicalName(reg.dim, reg._gmsh_id, reg.name)
+            reg_gmsh_id = gmsh.model.addPhysicalGroup(reg.dim, tags, tag=-1)
```

Comment:
reg._gmsh_id association to the GMSH Physical  group.
Original code is correct.